docs/conf.py

```diff
@@ -0,0 +1,35 @@
+project = 'Pangeo Docker Stacks'
+copyright = '2023, Pangeo Contributors'
+author = 'Pangeo Contributors'
+
+
+extensions = [
+    "myst_parser",
+]
+
+myst_enable_extensions = [
+    "deflist",
+    "colon_fence",
+]
```

Comment:
Add [linkify](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#linkify) perhaps? I'm seeing some URL links at https://pangeo-docker-images--473.org.readthedocs.build/en/473/howto/launch.html that don't seem to be clickable.
```suggestion
myst_enable_extensions = [
    "deflist",
    "colon_fence",
    "linkify",
]
```